CONTRIBUTING_JS.md

```diff
@@ -156,72 +144,27 @@ For releasing a js-ipfs, see [RELEASE_ISSUE_TEMPLATE](https://github.com/ipfs/js
 
 The commit message formatting can be added using a typical git workflow or through the use of a CLI wizard ([Commitizen](https://github.com/commitizen/cz-cli)).
 
-#### Commit Message Format
+- **Type** - Must be one of the following:
+  - **feat**: A new feature
+  - **fix**: A bug fix
+  - **docs**: Documentation only changes
+  - **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+  - **refactor**: A code change that neither fixes a bug nor adds a feature
+  - **perf**: A code change that improves performance
+  - **test**: Adding missing tests
+  - **chore**: Changes to the build process or auxiliary tools and libraries such as documentation generation
+- **Scope** - The scope could be anything specifying the place of the commit change. For example `api`, `cli`, etc...
+- **Breaking Changes** - Should be part of the commit message. Start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
 
-Each commit message consists of a header, a body and a footer. The header has a special format that includes a type, a scope and a subject:
-
-```
-<type>(<scope>): <subject>
-<BLANK LINE>
-<body>
-<BLANK LINE>
-<footer>
-```
-
-The header is mandatory and the scope of the header is optional.
-
-#### Revert
-
-If the commit reverts a previous commit, it should begin with `revert: `, followed by the header of the reverted commit. In the body it should say: `This reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
-
-#### Type
-
-Must be one of the following:
-
-* **feat**: A new feature
-* **fix**: A bug fix
-* **docs**: Documentation only changes
-* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
-  semi-colons, etc)
-* **refactor**: A code change that neither fixes a bug nor adds a feature
-* **perf**: A code change that improves performance
-* **test**: Adding missing tests
-* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
-  generation
-
-#### Scope
-
-The scope could be anything specifying the place of the commit change. For example `api`, `cli`, etc...
-
-#### Subject
-
-The subject contains a succinct description of the change:
-
-* use the imperative, present tense: "change" not "changed" nor "changes"
-* don't capitalize first letter
-* no dot (.) at the end
-
-#### Body
-
-Just as in the subject, use the imperative, present tense: "change" not "changed" nor "changes". The body should include the motivation for the change and contrast this with previous behavior.
-
-#### Footer
-
-The footer should contain any information about breaking changes and is also the place to reference GitHub issues that this commit closes.
-
-**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
-
-#### Examples
+Examples:
```

Comment:
Could you add an example with a breaking change message please?

Reply:
@achingbrain there is already one below ;)

Reply:
There's an example, but it's just a regular feature commit?

Reply:
See

<img width="801" alt="image" src="https://user-images.githubusercontent.com/1211152/75784618-c6b2b480-5d62-11ea-9fee-13e370e79566.png">